src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -251,6 +251,137 @@ auto createForceModel( PMB, Fracture, ParticleType particles,
         delta, K, G0, temp, alpha, temp0 );
 }
 
+template <typename TemperatureType>
+struct ForceModel<PMB, Elastic, DynamicTemperature, TemperatureType>
+    : public ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
+{
+    using base_type =
+        ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>;
+    using base_model = PMB;
+    using fracture_type = Elastic;
+    using thermal_type = DynamicTemperature;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    // Thermal parameters
+    using base_type::alpha;
+    using base_type::temp0;
+    double thermal_coeff;
+    double kappa;
+    double cp;
+
+    // Explicitly use the temperature-dependent stretch.
+    using base_type::thermalStretch;
+
+    // ForceModel(){};
+    ForceModel( const double _delta, const double _K,
+                const TemperatureType _temp, const double _kappa,
+                const double _cp, const double _alpha,
+                const double _temp0 = 0.0 )
```

Comment:
Remind me why do we need to set _temp0 = 0.0?

Reply:
We don't, it's a default value

Reply:
Does this mean it sets it to 0 if the user doesn't provide a value?

Reply:
Yes, it defaults to zero